docs/self-managed/setup/install.md

```diff
@@ -52,6 +52,23 @@ Refer to the [Operate](/self-managed/operate-deployment/operate-configuration.md
 
 When installing the [camunda-platform](https://artifacthub.io/packages/helm/camunda/camunda-platform) Helm chart, all components shown on the architectural diagram above are installed.
 
+## Licensing
```

Comment:
Unsure about the positioning of this section - right now, it sits above the Helm chart instructions, and isn't necessary/a prerequisite for installation. What if this sat beneath the Installation section and above [Configuring Enterprise components and Connectors](https://docs.camunda.io/docs/next/self-managed/setup/install/#configuring-enterprise-components-and-connectors)? The page outline would look like prereqs > installation > configure license > configure components.

Reply:
Adjusted in latest commit -- I am indeed wondering if we shouldn't add something to announcements if we are worried users might miss this, or wonder why the nav/logs are showing this now. Do you have any strong opinions?

Reply:
I added throughout the components in Self Managed, but let me know if you feel this is overkill.